CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 * [ENHANCEMENT] Ruler: Add more detailed query information to ruler query stats logging. #1411
 * [ENHANCEMENT] Admin: Admin API now has some styling. #1482 #1549
 * [ENHANCEMENT] Alertmanager: added `insight=true` field to alertmanager dispatch logs. #1379
+* [ENHANCEMENT] test: test: use `t.TempDir` to create temporary test directory. #1615
```

Comment:
We only include user-visible changes in the CHANGELOG. Since this PR only changes code, and is not user visible, let's remove this entry.

Reply:
Thanks for reviewing this PR. I have removed the changelog entry.